src/set_theory/game/pgame.lean

```diff
@@ -660,6 +660,102 @@ def relabel_relabelling {x : pgame} {xl' xr'} (el : x.left_moves ≃ xl') (er :
 
+/-- `is_option' x y` means that `x` is either a either a left or right option of `y`, up to
+relabelling. Contrast with `is_option`. -/
+@[mk_iff] inductive is_option' : pgame → pgame → Prop
+| relabelling_left {x₁ x₂ y : pgame} (h : is_option x₁ y) (e : x₁.relabelling x₂) : is_option' x₂ y
+
+theorem is_option.is_option' {x y : pgame} (h : is_option x y) : is_option' x y :=
+⟨h, relabelling.refl x⟩
+
+theorem is_option'.move_left {x : pgame} (i : x.left_moves) : is_option' (x.move_left i) x :=
+(is_option.move_left i).is_option'
+
+theorem is_option'.move_right {x : pgame} (j : x.right_moves) : is_option' (x.move_right j) x :=
+(is_option.move_right j).is_option'
+
+theorem is_option'.mk_left {xl xr : Type u} (xL : xl → pgame) (xR : xr → pgame) (i : xl) :
+  is_option' (xL i) (mk xl xr xL xR) :=
+@is_option'.move_left (mk _ _ _ _) i
+
+theorem is_option'.mk_right {xl xr : Type u} (xL : xl → pgame) (xR : xr → pgame) (j : xr) :
+  is_option' (xR j) (mk xl xr xL xR) :=
+@is_option'.move_right (mk _ _ _ _) j
+
+theorem is_option'.congr_left {x₁ x₂ y : pgame} (e : x₁.relabelling x₂) :
+  is_option' x₁ y ↔ is_option' x₂ y :=
+begin
+  split,
+  { rintro ⟨w, _, _, h, e'⟩,
+    exact ⟨h, e'.trans e⟩ },
+  { rintro ⟨w, _, _, h, e'⟩,
+    exact ⟨h, e'.trans e.symm⟩ },
+end
+
+theorem is_option'.relabelling_right {x y₁ y₂ : pgame} :
+  is_option x y₁ → y₁.relabelling y₂ → is_option' x y₂ :=
+begin
+  rintro (⟨_, i⟩ | ⟨_, j⟩) ⟨_, _, L, R, hL, hR⟩,
+  { exact ⟨is_option.move_left _, (hL i).symm⟩ },
+  { have := (hR (R j)).symm,
+    rw equiv.symm_apply_apply at this,
+    exact ⟨is_option.move_right _, this⟩ }
+end
+
+/-- Contrast with `is_option'_iff`, which asks for a relabelling of the left argument. -/
+theorem is_option'_iff' {x y₂ : pgame} :
+  x.is_option' y₂ ↔ ∃ {y₁ : pgame} (e : y₁.relabelling y₂), x.is_option y₁ :=
+begin
+  classical,
+  cases y₂ with zl zr zL zR,
+  split,
+  { rintro ⟨y, x, _, (⟨_, i⟩ | ⟨_, j⟩), e⟩,
+    { refine ⟨mk zl zr (λ i', if i' = i then x else zL i') zR,
+        ⟨equiv.refl _, equiv.refl _, λ i', _, λ j, relabelling.refl _⟩, _⟩,
+      { by_cases hi : i' = i,
+        { subst hi,
+          simpa using e.symm },
+        { simp [hi] } },
+      { convert is_option.mk_left (λ i', if i' = i then x else zL i') zR i,
+        rw if_pos rfl } },
+    { refine ⟨mk zl zr zL (λ j', if j' = j then x else zR j'),
+        ⟨equiv.refl _, equiv.refl _, λ j, relabelling.refl _, λ j', _⟩, _⟩,
+      { by_cases hi : j' = j,
+        { subst hi,
+          simpa using e.symm },
+        { simp [hi] } },
+      { convert is_option.mk_right zL (λ j', if j' = j then x else zR j') j,
+        rw if_pos rfl } } },
+  { rintro ⟨y, e, h⟩,
+    exact is_option'.relabelling_right h e }
+end
+
+theorem is_option'.congr_right {x y₁ y₂ : pgame} (e : y₁.relabelling y₂) :
+  is_option' x y₁ ↔ is_option' x y₂ :=
+begin
+  rw [is_option'_iff', is_option'_iff'],
+  exact ⟨λ ⟨w, e', h⟩, ⟨w, e'.trans e, h⟩, λ ⟨w, e', h⟩, ⟨w, e'.trans e.symm, h⟩⟩
+end
```

Comment:
It would make sense to split this into an `is_option'.imp_right` (which works with dot notation), and then build `is_option'_congr_right` (which doesn't) by just applying it twice. Same for the left version obviously